lib/process-services/task-list/components/task-list.component.ts

```diff
@@ -241,6 +251,7 @@ export class TaskListComponent extends DataTableSchema implements OnChanges, Aft
         this.loadTasksByState().subscribe(
             (tasks) => {
                 this.rows = this.optimizeNames(tasks.data);
+                this.rows = this.filterByDate(this.rows);
```

Comment:
you are filtering the result
               this.rows = this.filterByDate(this.rows);

but why the service should have dueBefore and dueAfter and use those value in the POST
http://adfdev.envalfresco.com/activiti-app/api-explorer.html#!/tasks/listTasksUsingPOST
the pagination is not working because you are mixing the backend totalSize with a client one


 this.rows = this.filterByDate(this.rows);
               this.selectTask(this.landingTaskId);
               this.success.emit(tasks);
               this.isLoading = false;
               this.pagination.next({
                   count: tasks.data.length,
                   maxItems: this.size,
                   skipCount: this.page * this.size,
                   totalItems: tasks.total
               });
you are asking before how many task do you have with these filters? (without passing the dueAfter and dueBefore) e.f 15. and after you say wait filter those 15 by date, so probably they will be less for instance 12 but the tasks.data.length still contains 15 
in the
 ```private createRequestNode() {```

you should add your dueBefore and dueAfter